src/lib/sb-file-uploader-hoc.jsx

```
@@ -6,6 +6,9 @@ import {connect} from 'react-redux';
 import log from '../lib/log';
 import sharedMessages from './shared-messages';
 import {setFileHandle, setProjectError} from '../reducers/tw';
+import {loadRecentProjects} from '../reducers/recent-projects';
+import {loadRecentProjects} from '../reducers/recent-projects';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'loadRecentProjects'. Consider to delete it or rename it.

'loadRecentProjects' is defined here:


(lint/suspicious/noRedeclare)